  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  
  
  # Position Controllers ---------------------------------------

  j_c1_rf_position_controller:
    type: effort_controllers/JointPositionController
    joint: j_c1_rf
    pid: {p: 100.0, i: 0.01, d: 10.0}

  j_thigh_rf_position_controller:
    type: effort_controllers/JointPositionController
    joint: j_thigh_rf
    pid: {p: 100.0, i: 0.01, d: 10.0}


  j_tibia_rf_position_controller:
    type: effort_controllers/JointPositionController
    joint: j_tibia_rf
    pid: {p: 100.0, i: 0.01, d: 10.0}








